tests/e2e/specs/subscriptions/merchant/merchant-subscriptions-renew-subscription.spec.js

```diff
@@ -69,16 +54,17 @@ describeif( RUN_SUBSCRIPTIONS_TESTS ).skip(
 				( el ) => el.innerText
 			);
 
-			await merchant.login();
+			await shopper.logout();
 		} );
 
 		afterAll( async () => {
 			// Delete the user created with the subscription
 			await withRestApi.deleteCustomerByEmail( customerBilling.email );
-			await merchant.logout();
```

Comment:
This should still be there instead of adding it to individual tests IMO, otherwise we rely on the order of execution of tests for this file so that last test passes for clean slate.

Reply:
Would it be better to explicity logout rather that waiting for teardown to handle it? See https://github.com/Automattic/woocommerce-payments/pull/4657#discussion_r959863817